kratos/processes/fast_transfer_between_model_parts_process.cpp

```diff
@@ -170,6 +176,16 @@ void FastTransferBetweenModelPartsProcess::TransferWithFlags()
             }
         }
 
+        if (num_geometries != 0 && (mEntity == EntityTransfered::ALL || mEntity == EntityTransfered::GEOMETRIES || mEntity == EntityTransfered::NODESANDGEOMETRIES)) {
+            const auto it_geom_begin = mrOriginModelPart.GeometriesBegin();
+            #pragma omp for
```

Comment:
Parallel utils.

Reply:
You cannot, there are criticals later

Reply:
OK. Now I see. The `critical` portion was not highlighted.